rkt.spec

```diff
@@ -34,179 +43,82 @@ BuildRequires: libmount-devel
+%systemd_preun %{name}-metadata
 
 %postun
-%systemd_postun_with_restart %{repo}-metadata
+%systemd_postun_with_restart %{name}-metadata
 
 %files
-%doc CONTRIBUTING.md DCO LICENSE MAINTAINERS README.md Documentation/*
-%{_bindir}/%{repo}
-%{_libexecdir}/%{repo}
-%{_unitdir}/%{repo}-metadata.s*
-
-%changelog
-* Mon Aug 03 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.7.0-3.git6dae5d5
-- built rkt commit#6dae5d5
-
-* Mon Aug 03 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.7.0-2.git6dae5d5
-- built rkt commit#6dae5d5
-
-* Sun Jul 19 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.7.0-1
-- New version: 0.7.0, built rkt         commit#c5e8cd5
-
-* Wed Jun 17 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.6.1-1
-- New version: 0.6.1, built rkt         commit#30cb88c
-
-* Wed Jun 10 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.6-5.git06bf23b
-- built rkt commit#06bf23b
-
-* Sun Jun 07 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.6-4.git7bf926e
-- built rkt commit#7bf926e
-
-* Tue Jun 02 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.6-3.git25b862d
-- built rkt commit#25b862d
-
-* Fri May 29 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.6-2.gited97885
-- built rkt commit#ed97885
-
-* Thu May 28 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.6-1
-- New version: 0.5.6, built rkt         commit#139af2b
-
-* Wed May 27 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.5-7.git5e95eac
-- built rkt commit#5e95eac
-
-* Sun May 24 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.5-6.gite5be761
-- built rkt commit#e5be761
-
-* Mon May 11 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.5-5.git724e49e
-- built rkt commit#724e49e
-
-* Mon May 11 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.5-4.git724e49e
-- built rkt commit#724e49e
-
-* Fri May 08 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.5-3.gitd61a4c5
-- built rkt commit#d61a4c5
-
-* Mon May 04 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.5-2.gitb1190d9
-- built rkt commit#b1190d9
+%doc CONTRIBUTING.md DCO LICENSE README.md Documentation/*
 
-* Mon May 04 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.5-1
-- New version: 0.5.5, built rkt         commit#b1190d9
+%{_bindir}/%{name}
+%{_libexecdir}/%{name}/stage1.aci
 
-* Wed Apr 29 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.4-3.git40ecb47
-- built rkt commit#40ecb47
+/usr/lib/systemd/system/%{name}-gc.timer
+/usr/lib/systemd/system/%{name}-gc.service
+/usr/lib/systemd/system/%{name}-metadata.socket
+/usr/lib/systemd/system/%{name}-metadata.service
 
-* Wed Apr 29 2015 Lokesh Mandvekar <lsm5@fedoraproject.org> - 0.5.4-2.git%{d_shortcommit}
-- built rkt commit#40ecb47
+%attr(0755,root,%{name}) %{_sharedstatedir}/%{name}
+%attr(0775,root,%{name}) %{_sharedstatedir}/%{name}/cas
+%attr(0775,root,%{name}) %{_sharedstatedir}/%{name}/cas/db
+%attr(0775,root,%{name}) %{_sharedstatedir}/%{name}/tmp
+%attr(0700,root,%{name}) %{_sharedstatedir}/%{name}/containers
```

Comment:
Yeah, that's the purpose. There are operations short of rkt run which are safe for non-root users, but after reading Dan's comments on the docker group, my tendency has changed to requiring sudo.  I'll remove those for now, we can add them back if requested and approved.